brambling/models.py

```diff
@@ -1979,6 +1979,11 @@ class SavedReport(models.Model):
     querystring = models.TextField()
 
 
+class ProcessedStripeEvent(models.Model):
+    stripe_event_id = models.CharField(max_length=35)
+    created = models.DateTimeField(auto_now_add=True)
```

Comment:
It seems like this model should have a couple more things, namely:

- whether this is the live or test API
- a unique constraint on the event id (and on whether it's live or test, if that's relevant)